tedana/resources/external_regressor_configs/Mot12_CSF.json

```diff
@@ -0,0 +1,27 @@
+{
+    "regress_ID": "Motion12_CSF",
+    "info": "Contains 6 columns for motion, 6 for first derivatives of motion, and 1 for a CSF ROI time series.",
+    "detrend": true,
```

Comment:
I don't love the idea of including extra processing to external regressors within tedana. I think stuff like that should be done by the user, and possibly offloaded to `fmripost-tedana` eventually.

Reply:
Oh sorry, my bad. I misunderstood what detrend was doing.